src/main/java/com/library/java_library_management/repository/GeneralModeRepository.java

```diff
@@ -0,0 +1,200 @@
+package com.library.java_library_management.repository;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.library.java_library_management.controller.Controller;
+import com.library.java_library_management.dto.BookInfo;
+import com.library.java_library_management.dto.JsonInfo;
+import com.library.java_library_management.status.BookStatus;
+
+import java.awt.print.Book;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class GeneralModeRepository implements Repository{
+    private static final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+    ObjectMapper objectMapper = new ObjectMapper();
+    FileControl fileControl = new FileControl();
```

Comment:
```suggestion
    private final ObjectMapper objectMapper = new ObjectMapper();
    private final FileControl fileControl = new FileControl();
```

인스턴스 변수 선언 및 초기화 시 private final 키워드 선언 및 생성자에서 처리하는 것을 권장합니다.